.github/workflows/backport-command.yml

```diff
@@ -0,0 +1,61 @@
+name: backport-command
+on:
+  repository_dispatch:
+    types: [backport-command]
+jobs:
+  backport:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install gh cli
+        run: |
+          curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg
+          echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+          sudo apt update
+          sudo apt install gh
+          gh auth login --with-token <<< "${{ secrets.ACTIONS_BOT_TOKEN }}"
+
```

Comment:
```suggestion
```
i don't think this is needed. the github cli is [preinstalled on all github hosted runners](https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows). you will have to add an `env` section for `GITHUB_TOKEN` to the step below that uses `gh`. see [docs](https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows).